### PR TITLE
Milestone 2: Hit the API

### PR DESCRIPTION
**Pull request by sneeu-leeu**
## In this milestone I :
- Used the API documentation to connect my app.
- Rewrote my HTML to be dynamically rendered.
- Write async JavaScript

### CODE REVIEW

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @sneeu-leeu, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Used arrow function.
:+1: Used `async` and `await` with the `fetch` method to send and receive data.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .hintrc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "formatters": ["stylish"],
+  "hints": [
+    "button-type",
+    "disown-opener",
+    "html-checker",
+    "meta-charset-utf-8",
+    "meta-viewport",
+    "no-inline-styles:error"
+  ]
+}
```
Comment (Dipeshtwis) [inline .hintrc]: - [x] I would recommend removing this file as you are no longer using `HTML` linters.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @sneeu-leeu, 

### Sorry for repeating this again. As, I have forgotten to add an inline comment previously. Take this comment and inline-comment as your review.

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Used arrow function.
:+1: Used `async` and `await` with the `fetch` method to send and receive data.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shaher-11 — APPROVED** [review]
#### Hello @Shaher-11,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle::muscle:

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Pull request merged**